contracts/contracts/PreConfCommitmentStore.sol

```diff
@@ -19,15 +19,15 @@ contract PreConfCommitmentStore is OwnableUpgradeable, UUPSUpgradeable {
 
     /// @dev EIP-712 Type Hash for preconfirmation commitment
     bytes32 public constant EIP712_COMMITMENT_TYPEHASH =
```

Comment:
revertingTxHashes should always be a csv string right? If so we should document this in contracts or add format validation  

Also it seems that revertingTxHashes represent multiple allowed reverting tx hashes. Wouldn't it only be relevant if `txnHash` for a preconf reverted? Hence we'd only need to post a boolean

Reply:
my second question may have to do with a lack of understanding of the current preconf structure

Reply:
It's the same as the txnhashes array. There can be mutliple txn hashes in a single bid